src/main/data/facilities.json

```diff
@@ -1224,6 +1224,16 @@
     "code": "DGB",
     "description": "DOLBY UK"
   },
+  {
+    "code": "DGC",
+    "contact": {
+      "address": "Via Carlo Pisacane 5, 65124, Pescara, Italy",
+      "email": "luca@digitmovies.com",
+      "name": "Luca Di Silverio"
+    },
+    "description": "Digitmovies Cinema",
+    "url": "http://www.digitmoviescinema.com"
```

Comment:
```suggestion
    "url": "https://www.digitmoviescinema.com"
```

Is HTTPS available?

Reply:
When I tried https it didn't work (gave a privacy error message).